src/lava/magma/core/model/py/type.py

```diff
@@ -12,4 +18,104 @@ class LavaPyType:
         type, ty.Type[PyInPort], ty.Type[PyOutPort], ty.Type[PyRefPort]
     ]
     d_type: type
-    precision: int = None  # If None, infinite precision is assumed
+    precision: str = None  # If None, infinite precision is assumed.
```

Comment:
Why not `math.inf` for infinite precision?

Reply:
Addressed by introducing precision class, see 96d55e369307c301015c073d3ccec2049e89b006.